metron-stellar/stellar-common/README.md

```diff
@@ -1377,7 +1377,7 @@ IS_EMAIL
 To run the Stellar Shell directly from the Metron source code, run a command like the following.  Ensure that Metron has already been built and installed with `mvn clean install -DskipTests`.
 ```
 $ mvn exec:java \
-   -Dexec.mainClass="org.apache.metron.stellar.common.shell.StellarShell" \
+   -Dexec.mainClass="org.apache.metron.stellar.common.shell.cli.StellarShell" \
```

Comment:
`StellarShell`, the main driver class for the CLI-based REPL, was moved to its own package since it is only used by the CLI-based REPL.   This separates it from the other core classes that are used by **both** the Zeppelin and CLI-based REPLs.